.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,96 @@
+name: "Bug report"
+description: "File a bug report in OJS, OMP, or OPS. Try our [Software Support forum](https://forum.pkp.sfu.ca/c/questions/5) if you cannot provide reproducible steps and technical specifications."
+
+title: "[Main Feature] | Workflow Section - Page, Description of bug"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for taking the time to fill out this bug report! Please update the title to describe your issue.

```

```suggestion
       Thanks for taking the time to fill out this bug report! Please update the title to describe your issue.

```
Heading formatting should generally not be used unless it's an actual content heading for accessibility reasons (e.g. headings should not used to make text larger only).